keadm/cmd/keadm/app/cmd/cmd_others.go

```diff
@@ -91,5 +93,8 @@ func NewKubeedgeCommand() *cobra.Command {
 
 	cmds.AddCommand(edge.NewEdgeRollback())
 
+	cmds.AddCommand(get.NewEdgeGet())
```

Comment:
Should we consider adding certs parent flags here for commands that the get pod and the restart pod?